support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -72,10 +72,16 @@ class DigitalSubscriptionController(
         val maybePromotionCopy = {
           landingCopyProvider.promotionCopy(promoCodes ++ defaultPromos, DigitalPack, "uk", orderIsAGift)
         }
+
+        val mainElement = assets.getSsrCacheContentsAsHtml(
+          divId = s"digital-subscription-checkout-page-$countryCode",
```

Comment:
Do we know what this ID is used for?

Reply:
Am guessing its a containing div for the page with an id in it. Doesnt appear to go very far in  `SSRContent` file`MainTypes.scala` however, have simply followed the pattern used elsewhere in same call in `Application.scala`.